conda_forge_tick/migrators/conda_forge_yaml_cleanup.py

```diff
@@ -0,0 +1,38 @@
+import os
+import typing
+from typing import Any
+from ruamel.yaml import YAML
+
+from conda_forge_tick.xonsh_utils import indir
+from conda_forge_tick.migrators.core import MiniMigrator
+
+if typing.TYPE_CHECKING:
+    from ..migrators_types import AttrsTypedDict
+
+
+class CondaForgeYAMLCleanup(MiniMigrator):
+    keys_to_remove = ['max_r_ver', 'max_py_ver', 'compiler_stack']
```

Comment:
Can you also add `min_py_ver` and `min_r_ver`?

Reply:
Sure!